pkg/network/render_test.go

```diff
@@ -380,3 +483,33 @@ func Test_getMultusAdmissionControllerReplicas(t *testing.T) {
 		})
 	}
 }
+
+func fillDefaults(conf, previous *operv1.NetworkSpec) {
+	FillDefaults(conf, previous, 1400)
+}
+
+func setupTestInfraAndBasicRenderConfigs(t *testing.T, prevType, nextType string) (
```

Comment:
NIT: Can we simplify the function to take `operv1.Network` as prev and next?
Then we could just makes copies of the args and skip the if logic.
example of new call:
`setupTestInfraAndBasicRenderConfigs(t, OpenShiftSDNConfig, OpenShiftSDNConfig)`